@Test public void testListNameNotFound() throws Exception {
fail("Should have thrown and NameNotFoundException");
}